tasks/php-rh.yml

```diff
@@ -48,11 +48,11 @@
   args:
     creates: /etc/opt/rh/rh-php{{ php_version }}/php.d/20-memcache.ini
 
-- name: "Set php.ini memory to 512"
+- name: "Set memory limit in php.ini"
   lineinfile:
     path: /etc/opt/rh/rh-php{{ php_version }}/php.ini
     regexp: '^memory_limit'
-    line: 'memory_limit = 512M'
+    line: 'memory_limit = {{ atom_pool_php_memory_limit }}'
```

Comment:
@replaceafill, we need to do some extra changes on `php.ini` and I found this PR. I think we should allow this modification in Ubuntu too and have different values for the pool and `php.ini`. Please, let me know your thoughts, I could take it if you want ;)

Reply:
@jraddaoui I discussed this with @mamedin and realized it's not the right approach for the issue. So I think we should close this one if that's OK with you (I forgot to close it a few weeks ago).

Reply:
Sounds good, I'll chat with @mamedin too, thanks!